### DISCUSSION TITLE
NUCELO_WL55 port assignment problem

### DISCUSSION THREAD

**Discussion by davefes** [micropython/micropython#13154] [Libraries & Drivers] [answered]
Error in my simple_rxtx.py:
```py
# PA6 = RF_TXEN, PA7 = RF_RXEN
PA6 = Pin('PA6', Pin.OUT)
PA7 = Pin('PA7', Pin.OUT)
```
```
Traceback (most recent call last):
  File "main.py", line 15, in <module>
  File "simple_rxtx.py", line 10, in <module>
ValueError: Pin(PA6) doesn't exist
MicroPython v1.22.0-preview.208.gb5449b0f0.dirty on 2023-12-08; NUCLEO-WL55 with STM32WL55JCI7
```

Thinking that because mpconfigboard.h for the NUCLEO_WL55 uses PA6 and PA7 for a SPI bus I commented-out the lines:
```py
// SPI buses, do NOT use this bus on the E77-400M22S
//#define MICROPY_HW_SPI1_NSS                     (pin_A4)
//#define MICROPY_HW_SPI1_SCK                     (pin_A5)
//#define MICROPY_HW_SPI1_MISO                    (pin_A6)
//#define MICROPY_HW_SPI1_MOSI                    (pin_A7)
```
but still get the error.  What else do I need to change?

Thanks.

**Comment by andrewleech** [comment, marked as answer]
I'm not sure why it's not finding the pin by name, it's listed in https://github.com/micropython/micropython/blob/master/ports/stm32/boards/NUCLEO_WL55/pins.csv

You can always access the pins directly like: `Pin.cpu.A6`

**Reply by davefes** [reply]
Thank you, that works.

I had looked at all the files in `/micropython/ports/stm32/boards/NUCLEO_WL55`
